### PR TITLE
Add Slack integration guide

### PR DESCRIPTION
**Pull request by jiaqi-fmc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,29 @@ FormaCloudClariSpend StackSet Deleted!
 Deleting the Stack...
 FormaCloudClariSpend Stack deleted!
 ```
+
+## Slack Integration
+
+1. Create two Slack channels: `formacloud-cost-report` and `formacloud-saving-actions`.
```
Comment (ShanForma) [inline README.md:81]:
```suggestion
1. Create two Slack channels for FormaCloud ClariSpend and Optima, such as `formacloud-clarispend` and `formacloud-optima`. The channel names don't really matter. You can choose your own names.
```

File: README.md [hunk 2]
```diff
@@ -75,3 +75,29 @@ FormaCloudClariSpend StackSet Deleted!
...
+2. Add FormaCloud team members to the Slack channels using Slack Connect, so they can assist with any requests.
+Members to add:
+```
+shan@formacloud.io
+weiqi@formacloud.io
+jiaqi@formacloud.io
+hannah@formacloud.io
+gideon@formacloud.io
+shazil@formacloud.io
+```
```
Comment (ShanForma) [inline README.md:92]: andi@formacloud.io

File: README.md [hunk 3]
```diff
@@ -75,3 +75,29 @@ FormaCloudClariSpend StackSet Deleted!
...
+
+3. Visit https://slack.formacloud.io or https://api.formacloud.io/slack/install/limited_scopes link to see the installation page.
+![image](https://github.com/forma-cloud/FormaCloud/assets/117554189/4fb77907-25be-4944-8c0f-ebe5195aa836)
+4. Click "Add to slack" and click "Allow".
+![image](https://github.com/forma-cloud/FormaCloud/assets/117554189/5be17695-b114-4185-9852-3e23d877ef2a)
+5. After the installation, go to Apps in Slack.
+![image](https://github.com/forma-cloud/FormaCloud/assets/117554189/079a3637-4f1f-4f97-bbfe-9aedf84fce57)
+6. Click on the app name on top left of this page.
+![image](https://github.com/forma-cloud/FormaCloud/assets/117554189/d0a9aa1c-dd8d-489f-9aad-d2341ecf9ab5)
+7. Click "Add this app to a channel" then choose the formacloud channels.
```
Comment (ShanForma) [inline README.md:102]: We need to do this for both channels.

File: README.md (hunk 2 above)
Comment (jiaqi-fmc) [reply README.md:93]: Done.

File: README.md (hunk 3 above)
Comment (jiaqi-fmc) [reply README.md]: Done.

**Review by ShanForma — APPROVED** [review]

**Pull request merged**